src/devices/xyzroe.ts

```diff
@@ -463,6 +467,30 @@ const definitions: DefinitionWithExtend[] = [
             ]);
         },
     },
+    {
+        zigbeeModel: ['ZigUSB_C6'],
+        model: 'ZigUSB_C6',
+        vendor: 'xyzroe',
+        description: 'Zigbee USB switch with monitoring',
+        extend: [
+            deviceEndpoints({endpoints: {1: 1, 2: 2, 3: 3}}),
+            identify(),
+            electricityMeter({
+                cluster: 'electrical',
+                attributes: 'both',
+                configureReporting: electroReporting,
+                endpointNames: ['1'],
+            }),
+            temperature({
+                reporting: tempReporting,
+            }),
+            onOff({powerOnBehavior: true, endpointNames: ['1'], description: '🔌 Controls the USB port'}),
```

Comment:
Please remove the emojis here, none of the other descriptions use it.

Reply:
This refers to the device itself, why can't I use the description that seems correct to me? I reworked the code so that it would use standard functions, ok I agree with that. But the description of the fields in the device is the decision of the device developer.

Reply:
The description is fine, it's just the emojis which make it non standard (and I would like to keep thing similar)